### PR TITLE
adding Bot API 6.1

### PR DESCRIPTION
**Pull request by mi3lix9**
I have added the types of Bot API 6.1

This is my first contribution, please tell me if I have to do something :)

### CODE REVIEW

**Review by mi3lix9 — COMMENTED** [review]
Fix: HapticFeedback is a void not a callback

**Comment by KnorpelSenf** [comment]
Will review soon, this is still on my radar

File: types/telegram-web-app/index.d.ts [hunk 1]
```diff
@@ -97,6 +131,22 @@ interface WebApp {
...
+     */
+    openLink(url: string): void
+    /**
+     * A method that opens a telegram link inside Telegram app. The Web App will be closed.
+     */
+    openTelegramLink(url: string): void
+    /**
+     * A method that opens an invoice using the link url. The Web App will receive the
+     *  event invoiceClosed when the invoice is closed. If an optional callback parameter was passed,
+     *  the callback function will be called and the invoice status will be passed as the first argument.
```
Comment (KnorpelSenf) [inline types/telegram-web-app/index.d.ts:147]:
```suggestion
     * event invoiceClosed when the invoice is closed. If an optional callback parameter was passed,
     * the callback function will be called and the invoice status will be passed as the first argument.
```

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
All changes look solid to me from a content point of view. I found one small formatting error. If it does not fix CI, you should maybe consider running the formatting tools of this repository so that everything looks consistent. For example, you can see how your comment lines are much longer than all other existing comments.

**Comment by KnorpelSenf** [comment]
Looking at the errors in CI, it seems indeed like you should run the formatter :)

**Review by KnorpelSenf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**